get_exports.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
FYI there is a job in jenkins called 3scale-api-python-tests to run integration tests. Obviously it does something similar, it has to to run the tests. It just doesn't create anything, only collecting information from existing deployment and objects.

Reply:
This is helping script for manual run of unit tests.